pwa-shell/package.json

```diff
@@ -9,6 +9,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.7.2",
     "@fortawesome/react-fontawesome": "^0.1.4",
     "bootstrap": "^4.3.1",
+    "history": "^4.9.0",
```

Comment:
Don't add `history` to the dependencies: that package is already a dependency of `react-router-dom`. Use `withRouter` HOC instead, and access the injected `history` object (see https://reacttraining.com/react-router/web/api/withRouter)